Lesson32/event-and-state-app/merve2/src/App.js

```diff
@@ -0,0 +1,38 @@
+import { Exercise1 } from './components/Exercise1';
+import { Exercise2 } from './components/Exercise2';
+import { Exercise3 } from './components/Exercise3';
+import Exercise4 from './components/Exercise4';
+import { Exercise4v2 } from './components/Exercise4';
+import { Exercise5 } from './components/Exercise5';
+import Exercise6 from './components/Exercise6';
```

Comment:
Half of the imports are named imports, half are defaults imports. When building your own component, try to pick one approach and follow it where possible.